htdocs/compta/accounting-files.php

```diff
@@ -175,6 +181,7 @@
 			$sql .= " WHERE datef between ".$wheretail;
 			$sql .= " AND t.entity IN (".$db->sanitize($entity == 1 ? '0,1' : $entity).')';
 			$sql .= " AND t.fk_statut <> ".Facture::STATUS_DRAFT;
+			if(!empty($projectid)) $sql .= " AND fk_projet = ".$db->sanitize($projectid);
```

Comment:
When the param in sql is an int, the sanitization is done like this:
`if(!empty($projectid)) $sql .= " AND fk_projet = ".((int) $projectid);`
db->sanitize is used to sanitize a string by escaping the ' so not enough to avoid sql injection if the param is not inside ' '.